variables.tf

```diff
@@ -19,3 +19,7 @@ variable "cidr" {
   description = "CIDR for the whole VPC"
```

Comment:
Rename `variable "cidr" {` to `variable "cidr_block" {`

Reply:
has been renamed